### PR TITLE
Add reference page for complete preseed YAML fields

### PR DESCRIPTION
**Pull request by activus-d**
This pull request solves [issue #17](https://github.com/canonical/open-documentation-academy/issues/17) of the Open Documentation Academy repository.

## Summary

The PR adds a new reference page documenting the complete fields of the LXD preseed YAML file, which is used for non-interactive configuration. It also updates the relevant sections in the  initialization and clustering how-tos to link to this new reference.

### CODE REVIEW

File: doc/reference/preseed_yaml_fields.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    name: lxdbr0
+    type: bridge
+    project: default
+
+storage_pools:
+  - config: {}
+    description: ""
+    name: default
+    driver: zfs
+
```
Comment (ru-fu) [inline doc/reference/preseed_yaml_fields.md:30]: Can you add `storage_volumes`?
This was added in https://github.com/canonical/lxd/pull/12426

**Review by ru-fu — COMMENTED** [review]
Thank you @activus-d - this looks great!

There might be some fields missing (I noticed `storage_volumes` - sorry, I missed that in the original issue ;) ). But even if we don't catch all of them now, this is a great improvement, and it'll be easy to add missing fields later.

**Review by ru-fu — APPROVED** [review]
Thanks a lot @activus-d !

@simondeziel or @tomponline , can you have a quick look if you spot anything obvious that is missing in the preseed file? As I wrote above, no big issue if we miss some details now, we can always update it later. :slightly_smiling_face: 

@activus-d Before we can actually merge this, you'll need to sign the CLA. See here:  https://ubuntu.com/legal/contributors

**Comment by activus-d** [comment]
@ru-fu I signed the CLA before pushing the last commit, but the check still failed. I have signed the form again.

**Comment by ru-fu** [comment]
> @ru-fu I signed the CLA before pushing the last commit, but the check still failed. I have signed the form again.

Thanks! Apparently it takes a bit for it to update, but the check passed now. :)

**Review by tomponline — APPROVED** [review]
Thanks!

**Pull request merged**